src/containers/CartContainer.tsx

```
@@ -0,0 +1,57 @@
+    }
+  }
+
+  return (
+    <div>
+      {cart.map((p) => (
+        <>
+          <p>{p.title}</p>
+          <p>{p.id}</p>
+          <p>{p.price}</p>
+          <button onClick={() => handleBuyButtonClick(p.id)}>購入</button>
+          <button onClick={() => handleRemoveButtonClick(p.id)}>削除</button>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)